.github/workflows/redis_test.yml

```diff
@@ -0,0 +1,36 @@
+name: Redis Test Task
+on:
+  workflow_dispatch:
+jobs:
+  redis-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f #ref v2
+      - name: Install redis
+        uses: shogo82148/actions-setup-redis@v1
+        with:
+          redis-version: "7.x"
+      - name: Install net tools
+        run: |
+          sudo apt-get update
+          #sudo apt-get install -y redis-tools redis-server
```

Comment:
I guess we can remove this one if not used

Reply:
Adjusted as I want to be able to test both options (redis install via aptget and redis install via gha)